.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.6.5
```

Comment:
agreed we should remove the Gemfile.lock since this is a library. I think this file should also be removed for the same reason. the gemspec already includes the required minimum ruby version.

Reply:
@mdeltito, done!